Subject: Patch from Ubuntu
From:
Explicitly state that ~/.pam_environment shall be read, and move
the pam_env call to the end so it's read also when $HOME is
encrypted.
Index: at.git/pam.conf
===================================================================
--- at.git.orig/pam.conf	2016-06-29 11:24:00.829830387 +0100
+++ at.git/pam.conf	2016-06-29 20:38:19.272379289 +0100
@@ -2,9 +2,9 @@
 # The PAM configuration file for the at daemon
 #
 
-auth	required	pam_env.so
 @include common-auth
 @include common-account
 session    required   pam_loginuid.so
 @include common-session-noninteractive
 session    required   pam_limits.so
+auth	required	pam_env.so user_readenv=1
